api/base/filters.py

```diff
@@ -218,7 +218,22 @@ def parse_query_params(self, query_params):
                                 'source_field_name': source_field_name
                             }
                         })
-
+                    elif source_field_name == 'is_preprint':
+                        op = 'eq' if value == 'false' else 'ne'
+                        query.get(key).update({
+                            field_name: {
+                                'op': op,
+                                'value': None,
+                                'source_field_name': 'preprint_file'
+                            }
+                        })
+                        query['_is_preprint_orphan'] = {
```

Comment:
A test would be a good way to verify that (e.g. `test_filter_preprint_true_excludes_orphans`, maybe `false_includes` as well if we want them to show up in that case).